nmt/utils/misc_utils.py

```diff
@@ -29,7 +29,7 @@
 
 
 def check_tensorflow_version():
-  min_tf_version = "1.4.0-dev"
+  min_tf_version = "1.4.0-rc1"
```

Comment:
We would like to keep the `master` tracking tf-nightly, so this should still be `dev`.

I will create a new branch when 1.4.0 is official :)